src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Backup/Blocks/backup_info.html.twig

```diff
@@ -0,0 +1,52 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="alert alert-info" role="alert">
+  {% if isHostMode %}
+    <h4>{{ 'How to restore a database backup'|trans({}, 'Admin.Advparameters.Notification') }}</h4>
+    <p>{{ 'If you need to restore a database backup, we invite you to subscribe to a [1][2]technical support plan[/2][/1].'|trans({'[1]': '<strong>',  '[/1]': '</strong>', '[2]': '<a class="_blank" href="https://addons.prestashop.com/support/16298-support-essentiel-plan.html">', '[/2]': '</a>'}, 'Admin.Advparameters.Notification')|raw }}</p>
+    <p>{{ 'Our team will take care of restoring your database safely.'|trans({}, 'Admin.Advparameters.Notification') }}</p>
+    <br>
+    <p>{{ 'Why can\'t I restore it by myself?'|trans({}, 'Admin.Advparameters.Notification') }}</p>
+    <p>{{ 'Your shop is hosted by PrestaShop. Although you can create backup files here below, there are core settings you cannot access for security reasons, like the database management. Thus, only our team can proceed to a restoration.'|trans({}, 'Admin.Advparameters.Notification') }}</p>
+  {% else %}
+    <h4>{{ 'How to restore a database backup in 10 easy steps'|trans({}, 'Admin.Advparameters.Help') }}</h4>
+    <ol class="mb-0">
+      <li>{{ 'Set "Enable Shop" to "No" in the "Maintenance" page under the "Preferences" menu.'|trans({}, 'Admin.Advparameters.Help') }}</li>
+      <li>{{ 'Download the backup from the list below or from your FTP server (in the folder "admin/backups").'|trans({}, 'Admin.Advparameters.Help') }}</li>
+      <li>{{ 'Check the backup integrity: Look for errors, incomplete file, etc... Be sure to verify all of your data.'|trans({}, 'Admin.Advparameters.Help') }}</li>
```

Comment:
In fact, it would be better written like this: _Check the backup integrity: look for errors, incomplete file, etc. Be sure to verify all of your data._ Perhaps it is too late as the string is already used elsewhere in the software.

Reply:
i could update it if needed. :)

Reply:
it's always needed, @LouiseBonnard is our queen :crown: